src/main/components/Browse/Dashboard.js

```diff
@@ -1,155 +1,106 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-import styled, { css } from "react-emotion";
-import { Card, CardHeader, Collapse, CardBody, CardTitle } from "reactstrap";
-import DASHBOARD_NAMES from "./dashboardNames.js";
+import { withStyles } from "@material-ui/core/styles";
+import Table from "@material-ui/core/Table";
+import TableBody from "@material-ui/core/TableBody";
+import TableCell from "@material-ui/core/TableCell";
+import TableHead from "@material-ui/core/TableHead";
+import TableRow from "@material-ui/core/TableRow";
+import Paper from "@material-ui/core/Paper";
 import AnalysisItem from "./Analysis.js";
 
-class DashboardBase extends Component {
+const styles = theme => ({
+  root: {
+    marginTop: theme.spacing.unit * 3,
+    width: "100%",
+    overflowX: "auto"
+  }
+});
+
+const CustomTableCell = withStyles(theme => ({
+  head: {
+    backgroundColor: "#686868",
+    color: "#FFFFFF",
+    fontSize: 15
+  }
+}))(TableCell);
+
+class Dashboard extends Component {
```

Comment:
After you remove the bind to handleAnalysisClick, you should switch this into a function.